modules/storages/app/workers/storages/copy_project_folders_job.rb

```diff
@@ -42,6 +42,7 @@ def perform(source:, target:, work_packages_map:)
       user = batch.properties[:user]
 
       project_folder_result = results_from_polling || initiate_copy(target)
+      project_folder_result.on_failure { |failed| return log_failure(failed) }
```

Comment:
❓ so, this is the actual fix, the rest is boyscouting, right? Or did I miss another piece of changes?

Reply:
Yep, this is the entire fix, the rest was just a case of "Ok, since I'm here already..."